client/modules/accounts/components/auth/loginButtons.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component, PropTypes } from "react";
+import {
+  Button,
```

Comment:
What's the limit then? 6 or more and we drop each onto it's own line? 5 or less we keep it on one?